nilearn/plotting/img_plotting.py

```diff
@@ -605,6 +618,8 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
             Specifies whether the colorbar should range from -vmax to vmax
             or from 0 to vmax. Setting to 'auto' will select the latter if
             the whole image is non-negative.
+        vmax: float
+            vmax passed to plt.imshow
```

Comment:
There appears to be a grammatical error here.

"function argument passed to" -> 231.000 hits on google
"function argument passed on to" -> 120.000.000 hits on google

Should we correct this in the entire nilearn code?